src/App.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div id="app">
-    <header-section />
+    <header-section v-if="!isEmbedded" />
     <div
       id="site-container"
       class="container"
```

Comment:
I think we should also remove the breadcrumbs in the embedded mode.

Reply:
Great decision. Done! 
<img width="897" alt="2021-03-05-181" src="https://user-images.githubusercontent.com/6351754/110131617-55a04580-7d98-11eb-95e5-93968e0dd88b.png">